### ISSUE TITLE
Utilise more screen width of messages

### ISSUE THREAD

**Issue by GitCurious** [FossifyOrg/Messages#215]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Messages/issues) or [closed](https://github.com/FossifyOrg/Messages/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Messages/discussions) or [closed](https://github.com/FossifyOrg/Messages/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Messages/blob/master/CONTRIBUTING.md).
- [X] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

[Optionally] Allow the user to utilise more of the screen with for messages, reducing wasted space either side of message, as hinted in red.

Also, possibly allowing to resize or relocate icon so it doesn't reduce message width further than is necessary.

![sms](https://github.com/user-attachments/assets/232e8d43-3a62-4290-8d55-e590638326cc)

### Why do you want this feature?

Reduce wasted space/more visible text

**Comment by naveensingh** [comment]
Yes, that can be done but I'm leaning towards the following change:

 - Increase message width to around ~90% of screen width. It's around 80% now.
 - Remove contact image for one-to-one conversations and reduce its size for group conversations.

**Comment by malev-v** [comment]
v 1.9.1
It still takes up space. 
Why not display the contact image first, followed by the message text? And leave a little white space on the right or left side.
And for short codes, it’s not needed at all. 
Right now, it just takes up space (the green bar).

<img alt="ignoreImageMinify" src="https://github.com/user-attachments/assets/08a8f0b4-d0b4-4aeb-ad21-38577e35d465" width=310 />
